packages/tools/src/eventListeners/annotations/contourSegmentation/areContoursFromSameSegmentIndex.ts

```diff
@@ -0,0 +1,24 @@
+import { ContourSegmentationAnnotation } from '../../../types/ContourSegmentationAnnotation';
+
+/**
+ * Check if two contour segmentations are from same segmentId,
+ * segmentationRepresentationUID and segmentIndex.
+ * @param firstAnnotation - First annotation
+ * @param secondAnnotation - Second annotation
+ * @returns True if they are from same segmentId, segmentationRepresentationUID
+ * and segmentIndex or false otherwise.
+ */
+export default function areContoursFromSameSegmentIndex(
```

Comment:
can we move to utilities.contourSegmentation.areSameSegments or something?

Reply:
Moved